pkg/config/defaults.go

```diff
@@ -27,6 +27,7 @@ import (
 
 // defaultConfig represents the default configuration for the DevWorkspace Operator.
 var defaultConfig = &v1alpha1.OperatorConfiguration{
+	EnableExperimentalFeatures: pointer.BoolPtr(false),
```

Comment:
> This should not be necessary (nil is to be interpreted as false).

Sounds good, will make the appropriate changes.

> Do we want to be able to enable experimental features in a DevWorkspaceOperatorConfig that isn't the main one?

IMO, no, we do not. In the current state of things (where Che owns its own DWOC) I don't think it makes sense to be able to toggle experimental features from Che, since usage of experimental features shouldn't really be encouraged. Enabling these features should be more of a "deeper" configuration option.

However, I think I understand the other point you're making:

> I think it's not a great idea, since the same option gates internal operator functionality, e.g. logging

I dislike the fact that enabling deeper logging (through `EnableExperimentalFeatures`) would also enable experimental features (e.g. PVC expansion). What if you just want more in-depth logging, without enabling experimental features?

This adds to the complexity of this change: do we update the DWOC API to have a new `verboseLoggingEnabled` option, to separate extra logging from `EnableExperimentalFeatures`? This would be a breaking API change.